### PR TITLE
Update to polkadot-v1.0.0

### PR DESCRIPTION
**Pull request by 1xstj**

### CODE REVIEW

File: proposals/src/header.rs [hunk 1]
```diff
@@ -47,11 +59,6 @@ pub struct ResourceId(pub [u8; 32]);
         scale_codec::MaxEncodedLen
     )
 )]
-#[cfg_attr(
-    feature = "std",
-    derive(Serialize, Deserialize),
-    serde(tag = "type", content = "id")
-)]
```
Comment (shekohex) [inline proposals/src/header.rs:54]: We still need these serde tags, it will be a breaking change if we removed these attr

**Review by shekohex — CHANGES_REQUESTED** [review]
Looks good, but I would say to regenerate the contracts and runtime and do not run cargo fmt on them, they are already formatted with another tool.

**Review by shekohex — APPROVED** [review]

**Comment by shekohex** [comment]
@1xstj could you also please change the CI for formatting to use rust nightly toolchain instead of the stable one? since we ignore all the generated files from the formatting

**Comment by 1xstj** [comment]
> @1xstj could you also please change the CI for formatting to use rust nightly toolchain instead of the stable one? since we ignore all the generated files from the formatting

not sure I follow, the CI failure is becuase we dont format the files in /contracts/protocol-solidity, how will nightly/stable change this? 🤔

**Comment by 1xstj** [comment]
Ran cargo fmt again, cannot merge PR without all checks being green

**Pull request merged**